Week-6/Exercise6_3.js

```diff
@@ -0,0 +1,20 @@
+function sortArr(arr) {
+
+    for (let i = 0; i < arr.length; i++) {
```

Comment:
considering complexity of this sorting, this solution will be not accepted. 
Read the question, array only contain 0s, 1s, and 2s, this array can be sorted in O(n).
Please think for O(n) solution.